Gopkg.toml

```diff
@@ -0,0 +1,34 @@
+# Gopkg.toml example
```

Comment:
No need to do it here now (as I know there's so much other tasks waiting as well), but an improvement proposal for future: This project would probably work pretty easily with go modules so worth to convert when suitable.